ensime-vars.el

```diff
@@ -314,9 +314,10 @@ Possible types: `organizeImport', `rename', `extractLocal',
   :type 'boolean
   :group 'ensime-ui)
 
-(defcustom ensime-use-helm nil
-  "Enable helm for some ensime features"
-  :type 'boolean
+(defcustom ensime-search-interface 'classic
```

Comment:
cool, but will breaks things for some people so we need to tweet / gitter about it.